vendor/assets/javascripts/react-chosen.js

```diff
@@ -0,0 +1,69 @@
+(function(window, React, $) {
```

Comment:
Also, why would we want different versions of react? If we pull in ui_components, we use the one it provides.